tests/components/device_tracker/test_owntracks.py

```diff
@@ -169,6 +190,15 @@ def test_location_update(self):
         self.assert_location_accuracy(60.0)
         self.assert_location_state('outer')
 
+    def test_location_inaccurate_gps(self):
+        self.hass.config.CONF_MAX_GPS_ACCURACY = 200
```

Comment:
I think your issue is that owntracks reads this value during `setup_scanner` - which has been run in the `setup_method` which is run before the test. 

So you could either change the core set-up with something like:-
````
    def setup_method(self, method):
        """ Init needed objects. """
        self.hass = get_test_home_assistant()
        mock_mqtt_component(self.hass)
        self.assertTrue(device_tracker.setup(self.hass, {
            device_tracker.DOMAIN: {
                CONF_PLATFORM: 'owntracks',
                CONF_MAX_GPS_ACCURACY: 200
            }}))
````
This will change the config for all tests, which I think is fine.